server/client.js

```diff
@@ -122,11 +120,7 @@ async function sendAPIKeyList(socket) {
     const timeLogger = new TimeLogger();
 
     let result = [];
-    const list = await R.find(
-        "api_key",
-        "user_id=?",
-        [ socket.userID ],
-    );
+    const list = await R.findAll("api_key");
```

Comment:
`api_key`s being at a user-namespace makes sense to me, think GitHub-PAT.
Is this change intentional?

Reply:
I don't know what is "GitHub-PAT", sorry 😕 

If I understand correctly how things works, this code is used in Settings > API keys. Based on that assomption, yes that change is intentional because every admin user should be able to manage them, like any other ressources.

Reply:
> GitHub-PAT

Ok "PAT" is for "Personal Access Token". I have already use that.
Thinking about that, the proper usage everyone should have is to only use in UK a "PAT" which is dedicated to notifications purpose. It's a kind of "system user" or "machine to machine" concept. I think that using a token associated to a personal user account is the worst thing to do. What happens if that user leave the boat at a time?

Reply:
> I think that using a token associated to a personal user account is the worst thing to do

I don't think so.
Assume a user leaves on bad terms and had access to said token.
Do you want said user to keep this access or revoke it at the same time as you revoke the users' account?

Yes, revoking credentials like this might break things, but this is likely infinitely better than leaking the rotated credentials of f. ex. your database.

Reply:
Perhaps I misunderstand what is API key in UK. AFAIK it's only kind of "credentials" used for notifications while using a third party service for that (ex: Slack). So if a user leaves on bad terms (worst scenario) he only have access to account dedicated to (most probably) internal notifications... so all he still can do is flood with undesirable message his ex-team... what a big deal! 🤣 

In addition, when a user leaves, the remaining people should disable his account in UK, then rotate all notifications dedicated API keys (usually not so many I guess). Looks like a basic security measure to change all credentials an ex-employee have access to when he leaves.

But again, perhaps I misunderstand what is API key in UK?

Reply:
> API-Keys allow access to the API (not a big concern at this point, but might be in the future) but also /metrics which allows leaking a lot of state that might be undesirable.

Okkkay!! Clearly, I misunderstand what is API keys in UK! I was thinking about a credential for third party system used for notification purpose (for ex: credentials to send a notification through Slack)! 🤦🏼‍♂️ 

Now I understand why you want to keep it per user and I'm perfectly aligned with you: API keys must of course stay user-scoped. So to answer your original question:
>Is this change intentional?

Not at all! 😨  I will revert that unwanted change right now!! 🏃🏼‍♂️ 
Thanks a lot to take time reviewing my code and give me so useful advise 🤩 😘

Reply:
Done! ✅